### DISCUSSION TITLE
Why i detected malware in VirusTotal?

### DISCUSSION THREAD

**Discussion by Mobinake** [lnreader/lnreader#724] [General]
This are the detected malware: 
¬ AhnLab-V3 -> PUP/Android.Malct.518314
¬ Avast-Mobile -> APK:RepMalware [Trj]

Is there no one monitoring the security and privacy? A well-intentioned question. Because i wanted to install the app.

**Comment by rajarsheechatterjee** [comment]
> AhnLab-V3 -> PUP/Android.Malct.518314

This is probably due to the APK being unsigned.

> Avast-Mobile -> APK:RepMalware [Trj]

This seems like a false positive introduced after adding a new library for a feature.

**Comment by rajarsheechatterjee** [comment]
A new release [v1.1.17](https://github.com/LNReader/lnreader/releases/tag/v1.1.17) has been published which removes the `APK:RepMalware [Trj]` issue.

**Comment by rajarsheechatterjee** [comment]
<img src="https://github.com/LNReader/lnreader/assets/57822170/989b1926-ad28-4749-a801-4fe9556c9f51" width="400px" />
